haiku-apps/fairtrade/fairtrade-1.0.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="The fair trade application supports sellers during the trade"
+DESCRIPTION="This application is designed for our local fair trade group (also see http://www.transfair.org/ (German link) for more info about the fair trade philosophy). The fair trade application supports sellers during the trade. Products can be scanned using a bar code scanner. All scanned products are showed in a shopping basket. For example, the seller scans a customer's product and the product appears in the shopping basket. If a customer wants to buy multiple products of the same kind, the product only has to be scanned once and the amount of products can be entered using the keypad. /
+Furthermore, it helps managing products on stock and provides an easy way to add new products. Sold products are automatically removed from the stock when completing a bargain. Product prices, descriptions, names and quantities can be edit as well. Old products can be disabled from the product view."
+HOMEPAGE="https://github.com/czeidler/fairtrade"
+COPYRIGHT="2006-2013 Clemens Zeidler"
+LICENSE="MIT"
+REVISION="1"
+commit="2b4ff648e2118021b092c4f0d9d0b61bc51c1374"
+SOURCE_URI="${HOMEPAGE}/archive/$commit.tar.gz"
+CHECKSUM_SHA256="8c4f411c008519c43fb5d4fa77867e883332fcea35c36fe9522223d3ec40695a"
```

Comment:
The chsckum we currently have in the recipe corresponds to the zip archive.
Since we swtiched to the tar.gz archive you have to update the checksum:
```
CHECKSUM_SHA256="3508410755c1bdf9c90403fd5bbf622f5f6036fe7a787cfe570c6542fd9f4186"
```

BTW, the build fails for me. I've tried so far fairtgrade and fairtrade_x86.